### ISSUE TITLE
Apache : log lines with URI containing spaces are not caught

### ISSUE THREAD

**Issue by benrubson** [fail2ban/fail2ban#1644]
Hello,

Apache URIs can contain spaces.
In this case, log lines are not caught by Fail2Ban.

I'll submit a PR to correct this 👍 

Ben

**Comment by szepeviktor** [comment]
Could you provide some error.log lines?

**Comment by benrubson** [comment]
Some examples showing that URIs can contain spaces :
```
[Sun Jan 01 21:41:10 2017] [error] [client 192.168.0.10] user bob not found: /backup/My Documents/
[Sun Jan 01 21:41:17 2017] [error] [client 192.168.0.10] user ben: authentication failure for "/backup/My Documents/": Password Mismatch
[Sun Jan 01 21:41:35 2017] [error] [client 192.168.0.10] client denied by server configuration: /var/www/archive/My Documents/
```

**Comment by szepeviktor** [comment]
Thanks.
It seem to be Apache 2.2.x

**Comment by benrubson** [comment]
Yep, but same with Apache 2.4 :
```
[Mon Jan 02 16:04:33.775458 2017] [authz_core:error] [pid 91974] [client 192168.0.10:62256] AH01630: client denied by server configuration: /var/www/data/My Documents
```

**Comment by sebres** [comment]
Fixed in #1645
